### PR TITLE
Fixed JSPB test failures

### PR DESCRIPTION
**Pull request by acozzette**

### CODE REVIEW

**Comment by acozzette** [comment]
@xfxyjwf

File: js/message_test.js [hunk 1]
```diff
@@ -1049,13 +1049,7 @@ describe('Message test suite', function() {
     var nested = new proto.jspb.test.Deeply.Nested.Message();
     nested.setCount(5);
     msg.setDeeplyNestedMessage(nested);
-
-    // After a serialization-deserialization round trip we should get back the
```
Comment (acozzette) [inline js/message_test.js]: I found that we don't currently generate `fromObject` methods (the code for it exists in js_generator.cc but is never called), so it seemed simplest to just avoid doing the round trip.

Comment (xfxyjwf) [inline js/message_test.js]: Does it still have the same test coverage? I thought the original bug can only be revealed after a round-trip. Is it not the case?

Comment (acozzette) [inline js/message_test.js]: You're right, I went ahead and added the binary serialization round trip back in.

**Pull request merged**